### ISSUE TITLE
Local echo for uploaded files

### ISSUE THREAD

**Issue by Velin92** [matrix-org/matrix-rust-sdk#2304]
realated to: https://github.com/vector-im/element-x-ios/issues/1288

**Comment by jplatte** [comment]
Soft-blocked on https://github.com/matrix-org/synapse/pull/15503.

**Comment by bmarty** [comment]
Duplicate of #1732?

**Comment by jplatte** [comment]
Yup.
